integration/kubernetes/runtimeclass_workloads/initContainer-shared-volume.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   initContainers:
   - name: first
     image: busybox
-    command: [ "sh", "-c", "date +%s > /volume/initContainer" ]
+    command: [ "sh", "-c", "date +%s > /volume/initContainer; sleep 2" ]
```

Comment:
Wouldn't `date +%s.%N` be better than `sleep`?

Reply:
Oh - I've just seen #2784 - why was that one closed?

Reply:
It's better than sleep, but the `date` command in `busybox` not support the `%N` option. I have tried in https://github.com/kata-containers/tests/pull/2784

Reply:
Gah - that's annoying! :smile: I guess you could check something like the interrupt count in `/proc/stat`, but I think I've found a better way:

```
$ docker run -ti busybox sh
/ # echo $EPOCHREALTIME
1597313595.934999
/ # echo $EPOCHREALTIME
1597313596.406377
/ #
```

Reply:
This is great. Let me try.